### ISSUE TITLE
When activating table shortcut key "T" to identify table, Narrator is not announcing table column count information

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#14059]
User Experience:
Users who rely on screen readers are impacted as Narrator is not announcing table column count information

Test Environment:
OS: Win 10
OS Version: 1909 (OS Build 18363.778)
Browser: Edge - Dev (Version 84.0.488.1)
URL: https://fabricweb.z5.web.core.windows.net/oufr/7.78.2/#/examples/detailslist
Screen reader : Narrator

Repro Steps:
Step 1: Turn on Narrator and open above URL using Edge Browser.
Step 2: Now activate table shortcut key "T"
Step 3: Verify the behavior of screen reader

Actual:
When activating table shortcut key "T" to identify table, Narrator is not announcing table column count information
Ex: It is announcing as "Data Grid Columns 4 rows"

Expected:
When activating table shortcut key "T" to identify table, Narrator should announce table column count information

Expected: Example - It should announce as "Call park Data Grid having 8 columns and 4 rows

![image](https://user-images.githubusercontent.com/53210862/87660748-327b4400-c77d-11ea-8812-3badf4a84f96.png)

**Comment by xugao** [comment]
@SyedRehmanRazvi - could you point me to the documentation for the shortcut T (for me to read up on what it is expected to do)? I don't find it [here](https://support.microsoft.com/en-us/help/22806/windows-10-narrator-keyboard-commands-touch-gestures).

**Comment by SyedRehmanRazvi** [comment]
> @SyedRehmanRazvi - could you point me to the documentation for the shortcut T (for me to read up on what it is expected to do)? I don't find it [here](https://support.microsoft.com/en-us/help/22806/windows-10-narrator-keyboard-commands-touch-gestures).

**Comment by SyedRehmanRazvi** [comment]
> @SyedRehmanRazvi - could you point me to the documentation for the shortcut T (for me to read up on what it is expected to do)? I don't find it [here](https://support.microsoft.com/en-us/help/22806/windows-10-narrator-keyboard-commands-touch-gestures).

Steps to reproduce the issue
1) Need to open https://fabricweb.z5.web.core.windows.net/oufr/7.78.2/#/examples/detailslist
2) Start Narrator (Screen reader) from windows start menu
3) Activate scan mode (using Caps Lock + Space)
4) Now press button "T" 
5) Now observe the Screen reader (Narrator) behavior

**Comment by xugao** [comment]
@SyedRehmanRazvi - thank you for the steps! I tried on a simple grid example on aria website [here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html), narrator is behaving the same. it's announcing every column by each's name. 
If you think this is still an issue, this needs to be addressed by Narrator team, not much we (components) can do here.
